es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
```

Comment:
Since we're no longer directly using Bootstrap Vue but replicating their BContainer, BRow, and BCol components ourselves, maybe it'd be more accurate to say "Based on" here.